backend/hdfs_backend.go

```diff
@@ -42,7 +43,7 @@ func (h *HdfsBackend) LatestVersion(checkForSuccess bool) (string, error) {
 	return "", fmt.Errorf("No valid versions at %s", h.displayURL(h.path))
 }
 
-func (h *HdfsBackend) Download(version string, destPath string) error {
+func (h *HdfsBackend) Download(version string, destPath string) (rterr error) {
```

Comment:
Fixed!

The named result param was so that the deferred cleanup func could return
an error if it encountered one -- only way to return something from a defer
-- but that moved to `download` in `sequins.go`, so this is not needed here
any longer.

On Wed, May 20, 2015 at 4:29 PM, Danielle Sucher <notifications@github.com>
wrote:

> In backend/hdfs_backend.go
> <https://github.com/stripe/sequins/pull/13#discussion_r30745496>:
>
> > @@ -42,7 +43,7 @@ func (h *HdfsBackend) LatestVersion(checkForSuccess bool) (string, error) {
> >  	return "", fmt.Errorf("No valid versions at %s", h.displayURL(h.path))
> >  }
> >
> > -func (h *HdfsBackend) Download(version string, destPath string) error {
> > +func (h *HdfsBackend) Download(version string, destPath string) (rterr error) {
>
> ooc, why name the result param here?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/stripe/sequins/pull/13/files#r30745496>.
>